homeassistant/components/roon/config_flow.py

```diff
@@ -0,0 +1,96 @@
+"""Config flow for Roon."""
+from asyncio import sleep
+from collections import OrderedDict
+import logging
+
+from roon import RoonApi
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_API_KEY, CONF_HOST
+from homeassistant.core import callback
+
+from .const import CONF_CUSTOM_PLAY_ACTION, DOMAIN, ROON_APPINFO
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@callback
+def configured_hosts(hass):
+    """Return a set of the configured hosts."""
+    return {
+        entry.data[CONF_HOST] for entry in hass.config_entries.async_entries(DOMAIN)
+    }
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class FlowHandler(config_entries.ConfigFlow):
+    """Handle a config flow."""
+
+    user_input = {}
```

Comment:
We shouldn't set mutable class attributes. That's dangerous. Please use our scaffold script to generate best practice config flow template code.